velox/exec/ContainerRowSerde.cpp

```diff
@@ -87,7 +87,7 @@ void serializeOne<TypeKind::ROW>(
   out.append<uint64_t>(nulls);
   for (auto i = 0; i < children.size(); ++i) {
     if (!bits ::isBitSet(nulls.data(), i)) {
-      serializeSwitch(*children[i], wrappedIndex, out);
+      serializeSwitch(*children[i]->loadedVector(), wrappedIndex, out);
```

Comment:
Can we do loadedVector() in caller sites such as deserializeArray()? lodedVector() is a function call that checks allLoaded_ flag inside LazyVector() not sure if it is costy on loaded vector? Also could you add a unit test for this? Thanks!